apps/client/src/pages/time-table/components/empty/empty-festival-section.tsx

```diff
@@ -0,0 +1,34 @@
+import { useNavigate } from 'react-router-dom';
+
+import { Button } from '@confeti/design-system';
+import { routePath } from '@shared/constants/path';
+import { IcFestivalGray } from '@confeti/design-system/icons';
+import * as styles from './empty-festival-section.css';
+
+const EmptyFestivalSection = () => {
+  const navigate = useNavigate();
+
+  const handleAddFestivalClick = () => {
+    navigate(routePath.SEARCH);
+  };
+
+  return (
+    <section className={styles.container}>
+      <div className={styles.iconDescriptionWrapper}>
+        <IcFestivalGray className={styles.icon} />
+        <h2 className={styles.description}>
+          페스티벌을 추가해
+          <br />
+          나만의 타임테이블을 등록해보세요
+        </h2>
```

Comment:
```suggestion
        <h1 className={styles.description}>
          페스티벌을 추가해
          <br />
          나만의 타임테이블을 등록해보세요
        </h1>
```
헤더가 없으니 페이지 제목으로 h1 !